### ISSUE TITLE
Display mining fee in status. Also display equivalent in user's preferred fiat currency.

### ISSUE THREAD

**Issue by wallclockbuilder** [bisq-network/bisq#5283]
### Description
![Screen Shot 2021-03-07 at 8 40 11 AM](https://user-images.githubusercontent.com/2033945/110249391-caa38480-7f2a-11eb-9374-6365aa45249e.png)
Fee rate displayed in bisq does not give much actionable information to the user. All we know is thats what the application will use. An improvement is to calculate and display to the user how much mining fees they will pay at the current fee rate, preferably in their preferred fiat currency.

### Expected behaviour
![Screen Shot 2021-03-07 at 8 55 31 AM](https://user-images.githubusercontent.com/2033945/110248011-e192a880-7f23-11eb-80a4-f414e4b9368d.png)
Bisq should show mining fee along with fee rate(which is useless without context).
The mining fee displayed needs to be the amount needed to get an offer's security deposit + fees onto the blockchain.

### Actual behaviour
![Screen Shot 2021-03-07 at 9 07 41 AM](https://user-images.githubusercontent.com/2033945/110249096-3b49a180-7f29-11eb-8014-2a7528a47c16.png)
Bisq displays mining fee only after take/make offer has been created

![Screen Shot 2021-03-07 at 9 07 09 AM](https://user-images.githubusercontent.com/2033945/110249164-aa26fa80-7f29-11eb-885e-bad627cf09dc.png)
You'll however notice that increasing the trade amount 1,000x(from 0.001BTC to 1BTC) does not change the mining fee much. It changed only by $0.38. 
This is proof that using the security deposit for calculating the trade mining fee is reliable enough for most people trading under the 0.25BTC limit. Truth is once you cross that limit you stop caring about mining fees which become a very tiny percentage of the trade amount.

**Comment by wallclockbuilder** [comment]
This same algorithm can be used by bisq to notify the user when the mining fee drops below $1. #5282

**Comment by wallclockbuilder** [comment]
![Screen Shot 2021-03-07 at 10 12 08 AM](https://user-images.githubusercontent.com/2033945/110250015-fc6a1a80-7f2d-11eb-9381-2dd80b8610a4.png)
This is inspired by mempool.space which displays the fee rate and a fiat friendly conversion.

**Comment by wallclockbuilder** [comment]
https://mempool.space

**Comment by wallclockbuilder** [comment]
Closing in favor of #5282

**Comment by pazza83** [comment]
I like this idea and #5280 

There are multiple fees though:

- Take offer
- Make offer
- Send BTC
- Send BSQ
- DAO transactions

**Comment by wallclockbuilder** [comment]
This solution is specifically for any transaction that is going to the bitcoin blockchain thus needs a low mining fee. Didn't  cross my mind but I guess people will use this notification as a prompt to not only create offers but also send BTC/BSQ and DAO votes and other DAO transactions that are bitcoin based. Wow. Powerful.

We may have to decouple the fee notification from the create offer form so people can use it to trigger ALL the bitcoin based transactions(not just offers).

**Comment by wallclockbuilder** [comment]
This will still need approval from dev team lead @ripcurlx before coding can begin.

**Comment by pazza83** [comment]
How about a 'fees' section in the app that can dynamically pull in the fee information for all uses based on the current sat/vB

**Comment by wallclockbuilder** [comment]
Walk me through how a user would interact with the fees section?

**Comment by pazza83** [comment]
Maybe:

Setting > Network Info

### Current Network fees

Here are the current network fees updated in real time. Please note estimates are not exact and may change depending on the number of UTXOs. The below fees are denominated in BTC and your default fiat currency:

Sat/vB: XX Sat/vB

**Trade fees**

Make offer fee: XX Sat/vB / $1.00
Take offer fee: XX Sat/vB / $3.00

**Send fees**

Send BTC fee: X Sat/vB / $1.00
Send BSQ fee: X Sat/vB / $1.35

**DAO transactions**

Fee: X Sat/vB / $1.35

**Comment by wallclockbuilder** [comment]
So far good. How does the user interact with the notification?

**Comment by wallclockbuilder** [comment]
> I like this idea and #5280
> 
> There are multiple fees though:
> 
>     * Take offer
> 
>     * Make offer
> 
>     * Send BTC
> 
>     * Send BSQ
> 
>     * DAO transactions

Take & Make offer fees are already covered by #5282. 
I'll create a separate issue for Send BTC fees & Send BSQ fees.
For DAO transactions I'm thinking Vote fees & BSQ Lockup fees. 
Looks to me like Burn BSQ fees don't matter here since if you're burning BSQ you won't care about the minining fee. Not sure. Haven't done any proof of burn transactions yet. 

What do you think @pazza83, do fees matter when doing a proof of burn? (thought it doesn't since you're in destruction mode.)

**Comment by wallclockbuilder** [comment]
Are there any other DAO transactions(outside of vote & lockUpBSQ) that are impacted by mining fees? I will like to include them in the low mining fee solution.

**Comment by wallclockbuilder** [comment]
I'm taking `Send BTC` fees off the list. 

![set n use custom mining fee](https://user-images.githubusercontent.com/2033945/110980730-066b8f00-831b-11eb-89e7-f6bf88550b70.png)
Bisq already has an option to set custom mining fees.

**Comment by pazza83** [comment]
> Bisq already has an option to set custom mining fees.

It does not show it in users preferred currency which would be nice and is your initial issue title :)
